<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPExample</class>
 <widget class="QMainWindow" name="TCPExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPExample</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="listen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Click Connect to connect to host, or listen to enter server mode... </string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstMessages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>disconnect</zorder>
   <zorder>listen</zorder>
   <zorder>connect</zorder>
   <zorder>send</zorder>
   <zorder>lblInfo</zorder>
   <zorder>lstMessages</zorder>
   <zorder>tcpText</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
